main.py

```diff
@@ -88,7 +92,7 @@ async def test_slash_command(interaction: discord.Interaction):
 watching.register_commands(tree, placetw_guild, client)
 
 # * register commands to the other servers
-for guild_id in GUILDS:
+for guild_id in GUILDS_DICT["guilds"].values():

```

Comment:
Instead of doing this for loop, we could do something like what we have in confessions with the list of guilds

Reply:
Though I suggest changing this later since we have to change EVERY module's `register_commands()` function basically

Reply:
yeah you need to change every `register_commands`